apps/checker/app/utils/JsonHelpers.scala

```diff
@@ -0,0 +1,10 @@
+package utils
+
+import java.net.URL
+import play.api.libs.json.{JsString, Json, Writes}
+
+object JsonHelpers {
+  implicit val urlWrites: Writes[URL] = (o: URL) => JsString(o.toString)
+  def toNDJson[T](serializable: T)(implicit tjs: Writes[T]) =
+    Json.toJson(serializable).toString() + 31.toChar
```

Comment:
Granted this is old code, but it's difficult to parse. Does 31 represent a \n character or such?

Reply:
Agreed! It's a record separator char – I've stored it in a variable to avoid a comment in [bf3efd8](https://github.com/guardian/typerighter/pull/382/commits/bf3efd82e624bc25505bea9962d0c53b90ea9dde).

Worth noting that there's another mistake here, in that this is _not_ ndjson, it's [json-seq!](https://en.wikipedia.org/wiki/JSON_streaming#Record_separator-delimited_JSON) There are a few flavours of streaming json, confusingly. Rather than rename in this PR, I'll address in the next PR or so (there are a few changes to this file coming that add extra streaming utils, and the refactor touches a few files which will all need rebasing.)